### PR TITLE
Tilt images dev server reloads on client

### PR DESCRIPTION
**Pull request by tildedave**

### CODE REVIEW

**Comment by tildedave** [comment]
ping,

File: server/init.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const Routes = require('./routes');
+const React = require('react');
+const ReactDOM = require('react-dom');
+const { Router } = require('react-router');
+
+if (typeof window === 'object') {
```
Comment (cohitre) [inline server/init.js]: i don't think we need the `if` since this is only loaded on the client.

**Comment by cohitre** [comment]
lgtm

File: server/init.js (hunk 1 above)
Comment (tildedave) [inline server/init.js]: good call

Comment (tildedave) [inline server/init.js]: I'll also put the script tag at the bottom of the body so I can remove the event listener.

Comment (tildedave) [inline server/init.js]: done

**Pull request merged**